evaluation/gorilla/run_infer.py

```diff
@@ -167,6 +167,9 @@ async def process_instance(
     dataset = prepare_dataset(
         dataset_df, output_file=output_file, eval_n_limit=args.eval_n_limit
     )
+    url = 'https://raw.githubusercontent.com/ShishirPatil/gorilla/main/eval/eval-scripts/codebleu/parser/my-languages.so'
+    response = requests.get(url)
+    with open('evaluation/gorilla/my-languages.so','wb') as f: f.write(response.content)
```

Comment:
Can you use `os.path.join(os.path.dirname(__file__), "my-language.so")` to calculate the path?